modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -1170,6 +1185,142 @@ func (suite *KeeperTestSuite) TestWriteUpgradeOpenChannel() {
 				suite.Require().Equal(mock.UpgradeVersion, channel.Version)
 				suite.Require().Equal(types.ORDERED, channel.Ordering)
 
+				// assert that NextSeqRecv is incremented to 2, because channel is now ORDERED
+				// seq updated in WriteUpgradeOpenChannel to latest sequence (one packet sent) + 1
+				seq, found = suite.chainA.GetSimApp().IBCKeeper.ChannelKeeper.GetNextSequenceRecv(ctx, path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
+				suite.Require().True(found)
+				suite.Require().Equal(uint64(2), seq)
+
+				// assert that NextSeqAck is incremented to 2 because channel is now ORDERED
+				seq, found = suite.chainA.GetSimApp().IBCKeeper.ChannelKeeper.GetNextSequenceAck(ctx, path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
+				suite.Require().True(found)
+				suite.Require().Equal(uint64(2), seq)
+
+				// Assert that state stored for upgrade has been deleted
+				upgrade, found := suite.chainA.GetSimApp().IBCKeeper.ChannelKeeper.GetUpgrade(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
+				suite.Require().Equal(types.Upgrade{}, upgrade)
+				suite.Require().False(found)
+
+				counterpartyUpgrade, found := suite.chainA.GetSimApp().IBCKeeper.ChannelKeeper.GetCounterpartyUpgrade(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
+				suite.Require().Equal(types.Upgrade{}, counterpartyUpgrade)
+				suite.Require().False(found)
+
+				// events := ctx.EventManager().Events().ToABCIEvents()
```

Comment:
I noticed that some events testing is commented out in other test handlers. Is there an issue for these to be addressed?